### PR TITLE
feat: support multiple lark endpoint

### PR DESCRIPTION
**Pull request by tianzhou**

### CODE REVIEW

**Review by unknwon — APPROVED** [review]
Approve to unblock, use your judgement

File: main.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ import (
 
 func main() {
 	refPrefix := flag.String("ref-prefix", "refs/heads/", "The prefix for the ref")
-	larkURL := flag.String("lark-url", os.Getenv("LARK_URL"), "The Lark webhook URL")
+	larkURL := flag.String("lark-url", os.Getenv("LARK_URL"), "A comma separated list of Lark webhook URL")
```
[the remark this answers is not in the document]
Comment (tianzhou) [reply main.go]: Done

File: main.go [hunk 2]
```diff
@@ -51,10 +51,15 @@ func main() {
...
 
-		err = sendToLark(r.Context(), *larkURL, fmt.Sprintf("New commits have been pushed to %q: %s", payload.Ref, payload.Compare))
-		if err != nil {
-			return http.StatusInternalServerError, fmt.Sprintf("Failed to send to Lark: %v", err)
+		urlList := strings.Split(*larkURL, ",")
+
+		for _, url := range urlList {
+			err = sendToLark(r.Context(), url, fmt.Sprintf("New commits have been pushed to %q: %s", payload.Ref, payload.Compare))
+			if err != nil {
+				return http.StatusInternalServerError, fmt.Sprintf("Failed to send to Lark %q: %v", url, err)
```
[the remark this answers is not in the document]
Comment (tianzhou) [reply main.go]: Done

**Review by unknwon — COMMENTED** [review]
LGTM

**Pull request merged**